### PR TITLE
Statistics

### PR DESCRIPTION
**Pull request by UrbanAndrius**

### CODE REVIEW

File: MainForm.cs [hunk 1]
```diff
@@ -72,21 +72,26 @@ private void ChooseImage(object sender, EventArgs e)
...
-            this.statisticsChart.Series.Add(new Series("Data"));
-           
-            this.statisticsChart.Series["Data"]["PieLabelStyle"] = "Outside";
-            this.statisticsChart.Series["Data"].ChartType = SeriesChartType.Pie;
-            this.statisticsChart.Series["Data"].IsVisibleInLegend = false;
-            //this.statisticsChart.Series["Data"].Points.DataBindXY(
-            //    data.receipts.Select(item => item.Key).ToArray(),
-            //    data.receipts.Select(item => item.Value).ToArray()
-            //);
+            //AddChartSeries here
```
Comment (tomasmuzas) [inline MainForm.cs]: Isn't function call missing here?

Comment (UrbanAndrius) [reply MainForm.cs]: I would like you to add this line because I cannot find receipt object with List<Item>

Comment (tomasmuzas) [reply MainForm.cs]: Alright, will fix it

**Comment by tomasmuzas** [comment]
Everythins is fixed now :)

**Pull request merged**